docs/modules/ROOT/pages/how-to/long-pod-startup.adoc.adoc

```diff
@@ -0,0 +1,57 @@
+= CreateContainerError - Long Pod Startup
+
+This page will describe how to mitigate a very long pod startup time and `CreateContainerError`.
+
+== Explanation
+
+When a Pod has a volume with many files attached, the Pod startup time can be very long.
+
+This is caused by SELinux relabeling all the files in the volume. Depending on the number of files, the relabeling can take a long time.
```

Comment:
```suggestion
When a Pod mounts a volume which contains many files, the Pod startup time can be very long.
This is caused because the container runtime updates the SELinux labels of all the files in the volume on pod startup.
Depending on the number of files, the relabeling can take a long time.
```